modules/ROOT/pages/govern-api-instances.adoc

```diff
@@ -0,0 +1,45 @@
+= Governing API Instances
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+API Manager is closely integrated with xref:api-governance::index.adoc[API Governance] to govern aspects of API instances with governance rulesets. Applying a governance ruleset to your API ensures that every instance of that API conforms to standards such as what policies are applied and if a TLS context is required.  
```

Comment:
```suggestion
API Manager is closely integrated with xref:api-governance::index.adoc[Anypoint API Governance] to govern aspects of API instances with governance rulesets. Applying a governance ruleset to your API ensures that every instance of that API conforms to standards such as what policies are applied and whether a TLS context is required.  
```

Reply:
I recently learned that the only acceptable form of the product name is Anypoint API Governance. I missed this on the [Product Tiering and Hierarchy sheet](https://docs.google.com/spreadsheets/d/1f6v9cjx84ZAHGeZs36A0clZoQc_ggAe6m6e-NQ9BUuk/edit#gid=71218867), and no one on our team seemed to see it either. I'm not sure how changes there are supposed to be communicated, but now they're added to Vale, so that's how I learned about it. Refs to UI as API Governance are okay where that is what it's called there, or at least that is how I'm handling it.